### ISSUE TITLE
Building of 4.7.1 tag fails on IDE Stack: "Missing header in: che-core-ide-stacks/src/main/resources/images/type-bitnami.svg"

### ISSUE THREAD

**Issue by mariosotil** [eclipse-che/che#2389]
It failed on Mac OS X El Capitan and Ubuntu 16.04. Both have installed Maven 3.3.9

Steps to reproduce

```
$ git fetch
$ git checkout 4.7.1
$ mvn clean install
```

This is the error

```
[INFO] --- license-maven-plugin:2.10:check (check-headers) @ che-core-ide-stacks ---
[INFO] Checking licenses...
[WARNING] Missing header in: /home/ubuntu/.local/share/Yatta/Launcher/setup-64/Setup/che/ide/che-core-ide-stacks/src/main/resources/images/type-bitnami.svg
```

This is the log

```
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building Che Core :: IDE :: STACKS 4.7.1
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.6.1:clean (default-clean) @ che-core-ide-stacks ---
[INFO] 
[INFO] --- build-helper-maven-plugin:1.9.1:parse-version (parse-version) @ che-core-ide-stacks ---
[INFO] 
[INFO] --- buildnumber-maven-plugin:1.3:create (default) @ che-core-ide-stacks ---
[INFO] 
[INFO] --- maven-enforcer-plugin:1.4:enforce (enforce-java-version) @ che-core-ide-stacks ---
[INFO] 
[INFO] --- maven-enforcer-plugin:1.4:enforce (enforce-maven-version) @ che-core-ide-stacks ---
[INFO] 
[INFO] --- maven-enforcer-plugin:1.4:enforce (distribution-management-used) @ che-core-ide-stacks ---
[INFO] 
[INFO] --- maven-enforcer-plugin:1.4:enforce (no-distribution-management-at-all) @ che-core-ide-stacks ---
[INFO] 
[INFO] --- maven-enforcer-plugin:1.4:enforce (dependency) @ che-core-ide-stacks ---
[INFO] 
[INFO] --- maven-enforcer-plugin:1.4:enforce (ban-bad-dependencies) @ che-core-ide-stacks ---
[INFO] 
[INFO] --- maven-sortpom-plugin:2.3.0:verify (default) @ che-core-ide-stacks ---
[INFO] Verifying file /home/ubuntu/.local/share/Yatta/Launcher/setup-64/Setup/che/ide/che-core-ide-stacks/pom.xml
[INFO] 
[INFO] --- license-maven-plugin:2.10:check (check-headers) @ che-core-ide-stacks ---
[INFO] Checking licenses...
[WARNING] Missing header in: /home/ubuntu/.local/share/Yatta/Launcher/setup-64/Setup/che/ide/che-core-ide-stacks/src/main/resources/images/type-bitnami.svg
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Che Core :: API :: Core ............................ SUCCESS [ 35.582 s]
[INFO] Che Core :: API :: Model ........................... SUCCESS [  3.912 s]
[INFO] Che Master Parent .................................. SUCCESS [  0.084 s]
[INFO] Che Core :: API :: Authentication .................. SUCCESS [ 12.251 s]
[INFO] Che Core :: API :: Machine :: Shared ............... SUCCESS [ 11.166 s]
[INFO] Che Core :: API :: Workspace :: Shared ............. SUCCESS [  8.981 s]
[INFO] Che Core :: API :: Project Templates :: Shared ..... SUCCESS [  6.508 s]
[INFO] Che Core :: API :: Project Templates ............... SUCCESS [  4.434 s]
[INFO] Che Core :: API :: Machine ......................... SUCCESS [  9.524 s]
[INFO] Che Core :: API :: Workspace ....................... SUCCESS [ 20.567 s]
[INFO] Che Core :: API :: User :: Shared .................. SUCCESS [  5.724 s]
[INFO] Che Core :: API :: User ............................ SUCCESS [  9.647 s]
[INFO] Che Core :: API :: Factory :: Shared ............... SUCCESS [  7.532 s]
[INFO] Che Core :: API :: Factory ......................... SUCCESS [ 11.359 s]
[INFO] Che Core :: API :: SSH :: Shared ................... SUCCESS [  3.514 s]
[INFO] Che Core :: API :: SSH ............................. SUCCESS [  7.277 s]
[INFO] Che Core :: API :: Impl Local ...................... SUCCESS [  9.050 s]
[INFO] Che Agent Parent ................................... SUCCESS [  0.049 s]
[INFO] Che Core :: API :: Project Shared .................. SUCCESS [  7.632 s]
[INFO] Che Core :: API :: Agent ........................... SUCCESS [  4.439 s]
[INFO] Che Core :: API :: Project ......................... SUCCESS [ 49.208 s]
[INFO] Che Core :: API :: Git :: Shared ................... SUCCESS [  4.204 s]
[INFO] Che Core :: API :: Git ............................. SUCCESS [ 15.123 s]
[INFO] Che Plugin :: Parent ............................... SUCCESS [  0.048 s]
[INFO] Che Plugin :: SSH Key :: Parent .................... SUCCESS [  0.038 s]
[INFO] Che Plugin :: SSH Key :: Server .................... SUCCESS [  5.126 s]
[INFO] Che Core :: Git Impl JGit .......................... SUCCESS [ 15.913 s]
[INFO] Che Core :: API :: Debug :: Shared ................. SUCCESS [  4.460 s]
[INFO] Che Core :: API :: Debug ........................... SUCCESS [  9.228 s]
[INFO] Che Core :: IDE .................................... SUCCESS [  0.051 s]
[INFO] Che Core :: Commons :: GWT ......................... SUCCESS [ 14.655 s]
[INFO] Che Core :: IDE :: API ............................. SUCCESS [ 49.489 s]
[INFO] Che Core :: IDE :: Generators ...................... SUCCESS [  4.815 s]
[INFO] Che Core :: IDE :: UI .............................. SUCCESS [ 18.514 s]
[INFO] Che Core :: IDE :: App ............................. SUCCESS [ 45.221 s]
[INFO] Che Core :: IDE :: TEMPLATES ....................... SUCCESS [  0.042 s]
[INFO] Che Core :: IDE :: STACKS .......................... FAILURE [  0.025 s]
[INFO] Che Plugin ::  Product Info :: Client .............. SKIPPED
[INFO] Che Plugin :: Help :: Parent ....................... SKIPPED
[INFO] Che Plugin :: Help :: Client ....................... SKIPPED
[INFO] Che Plugin :: Docker :: Parent ..................... SKIPPED
[INFO] Che Plugin :: Docker :: Docker Client .............. SKIPPED
[INFO] Che Plugin :: Docker :: Machine .................... SKIPPED
[INFO] Che Plugin :: Web :: Parent ........................ SKIPPED
[INFO] Che Plugin :: Web :: File Types .................... SKIPPED
[INFO] Che Plugin :: Orion :: Parent ...................... SKIPPED
[INFO] Che Plugin :: Orion :: Editor ...................... SKIPPED
[INFO] Che Plugin :: Machine :: Parent .................... SKIPPED
[INFO] Che Plugin :: Machine .............................. SKIPPED
[INFO] che-plugin-machine-ext-server ...................... SKIPPED
[INFO] Che Plugin :: Machine :: SSH Client ................ SKIPPED
[INFO] Che Plugin :: Debugger :: Parent ................... SKIPPED
[INFO] Che Plugin :: Debugger :: Extension Debugger IDE ... SKIPPED
[INFO] Che Plugin :: Java :: Parent ....................... SKIPPED
[INFO] Che Plugin :: Java :: Extension JDT ................ SKIPPED
[INFO] Che Plugin :: Java :: Eclipse JFace ................ SKIPPED
[INFO] Che Plugin :: Java :: Eclipse JFace Text ........... SKIPPED
[INFO] Che Plugin :: Java :: org.eclipse.core.filesystem .. SKIPPED
[INFO] Che Plugin :: Java :: org.eclipse.core.resources ... SKIPPED
[INFO] Che Plugin :: Java :: org.eclipse.core.filebuffers . SKIPPED
[INFO] Che Plugin :: Java :: Eclipse Search ............... SKIPPED
[INFO] Che Plugin :: Java :: Eclipse IDE UI ............... SKIPPED
[INFO] Che Plugin :: Java :: Extension Java Shared ........ SKIPPED
[INFO] Che Plugin :: Java :: Eclipse LTK Refactoring ...... SKIPPED
[INFO] Che Plugin :: Java :: Eclipse JDT UI ............... SKIPPED
[INFO] Che Plugin :: Java :: Extension Java Server ........ SKIPPED
[INFO] Che Plugin :: Java :: Plain ........................ SKIPPED
[INFO] Che Plugin :: Java :: Plain :: Shared .............. SKIPPED
[INFO] Che Plugin :: Java :: Plain :: IDE ................. SKIPPED
[INFO] Che Plugin :: Orion :: Compare Widget .............. SKIPPED
[INFO] Che Plugin :: Java :: Extension Java Client ........ SKIPPED
[INFO] Che Plugin :: Java :: Plain :: Server .............. SKIPPED
[INFO] Che Plugin :: Java Debugger :: Parent .............. SKIPPED
[INFO] Che Plugin :: Java :: Java Debugger Server ......... SKIPPED
[INFO] Che Plugin :: Java :: Java Debugger IDE ............ SKIPPED
[INFO] Che Plugin :: Maven :: Parent ...................... SKIPPED
[INFO] Che Plugin :: Maven :: Maven Server ................ SKIPPED
[INFO] Che Plugin :: Maven :: Maven Server API ............ SKIPPED
[INFO] Che Plugin :: Maven :: Maven Server Impl ........... SKIPPED
[INFO] Che Plugin :: Maven :: Maven Tools ................. SKIPPED
[INFO] Che Plugin :: Maven :: Archetype Generator ......... SKIPPED
[INFO] Che Plugin :: Maven :: Extension Maven Shared ...... SKIPPED
[INFO] Che Plugin :: Maven :: Extension Maven Client ...... SKIPPED
[INFO] Che Plugin :: Maven :: Extension Maven Server ...... SKIPPED
[INFO] Che Plugin :: GDB :: Parent ........................ SKIPPED
[INFO] Che Plugin :: GDB :: GDB Server .................... SKIPPED
[INFO] Che Plugin :: GDB :: GDB IDE ....................... SKIPPED
[INFO] Che Plugin :: SDK :: Parent ........................ SKIPPED
[INFO] Che Plugin :: SDK :: Plugins Development ........... SKIPPED
[INFO] Che Plugin :: SDK :: Local Env ..................... SKIPPED
[INFO] Che Plugin :: SDK :: Tools ......................... SKIPPED
[INFO] Che Plugin :: Git :: Parent ........................ SKIPPED
[INFO] Che Plugin :: Git :: Extension ..................... SKIPPED
[INFO] Che Plugin :: Github :: Parent ..................... SKIPPED
[INFO] Che Plugin :: Github :: OAuth2 ..................... SKIPPED
[INFO] Che Plugin :: Github :: Credential provider ........ SKIPPED
[INFO] Che Plugin :: Github :: Shared ..................... SKIPPED
[INFO] Che Plugin :: SSH Key :: Client .................... SKIPPED
[INFO] Che Plugin :: Github :: IDE ........................ SKIPPED
[INFO] Che Plugin :: Github :: Server ..................... SKIPPED
[INFO] Che Plugin :: Dashboard :: Parent .................. SKIPPED
[INFO] Che Plugin :: Dashboard :: Client .................. SKIPPED
[INFO] Che Plugin :: GWT :: Parent ........................ SKIPPED
[INFO] Che Plugin :: GWT .................................. SKIPPED
[INFO] Che Plugin :: Python :: Parent ..................... SKIPPED
[INFO] che-plugin-python-lang-shared ...................... SKIPPED
[INFO] Che Plugin :: Python :: Extension Server ........... SKIPPED
[INFO] Che Plugin :: Python :: IDE ........................ SKIPPED
[INFO] Che Plugin :: SVN :: Parent ........................ SKIPPED
[INFO] Che Plugin :: SVN :: Shared ........................ SKIPPED
[INFO] Che Plugin :: SVN :: Server ........................ SKIPPED
[INFO] Che Plugin :: SVN :: IDE ........................... SKIPPED
[INFO] Che Plugin :: C/C++ :: Parent ...................... SKIPPED
[INFO] Che Plugin :: C/C++ :: Shared ...................... SKIPPED
[INFO] Che Plugin :: C/C++ :: Extension Server ............ SKIPPED
[INFO] Che Plugin :: C/C++ :: IDE ......................... SKIPPED
[INFO] Che Plugin :: NodeJs :: Parent ..................... SKIPPED
[INFO] Che Plugin :: NodeJs :: Extension Shared ........... SKIPPED
[INFO] Che Plugin :: NodeJs :: IDE ........................ SKIPPED
[INFO] Che Plugin :: NodeJs :: Extension Server ........... SKIPPED
[INFO] Che Plugin :: Ssh machine .......................... SKIPPED
[INFO] Che Sample :: Parent ............................... SKIPPED
[INFO] Che Sample :: Plugin JSON :: Parent ................ SKIPPED
[INFO] Che Sample :: Plugin JSON :: Shared ................ SKIPPED
[INFO] Che Sample :: Plugin JSON :: IDE ................... SKIPPED
[INFO] Che Sample :: Plugin JSON :: Server ................ SKIPPED
[INFO] Che Sample :: Plugin FileType :: Parent ............ SKIPPED
[INFO] Che Sample :: Plugin FileType :: IDE ............... SKIPPED
[INFO] Che Sample :: Plugin Wizard :: Parent .............. SKIPPED
[INFO] Che Sample :: Plugin Wizard :: Shared .............. SKIPPED
[INFO] Che Sample :: Plugin Wizard :: Server .............. SKIPPED
[INFO] Che Sample :: Plugin Wizard :: IDE ................. SKIPPED
[INFO] Che Dashboard :: Web App ........................... SKIPPED
[INFO] Che IDE :: Parent .................................. SKIPPED
[INFO] Che Workspace Agent War Packaging .................. SKIPPED
[INFO] Che Assembly Workspace Agent Server ................ SKIPPED
[INFO] Che IDE :: Compiling GWT Application ............... SKIPPED
[INFO] Che IDE :: Compiling WS Master WAR ................. SKIPPED
[INFO] Che IDE :: Assemblies Tomcat ....................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 06:59 min
[INFO] Finished at: 2016-09-08T21:41:22-04:00
[INFO] Final Memory: 281M/1677M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.mycila:license-maven-plugin:2.10:check (check-headers) on project che-core-ide-stacks: Some files do not have the expected license header -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :che-core-ide-stacks
```

**Comment by mariosotil** [comment]
**Workaround:** Build Che with this command

```
$ mvn clean install -Dlicense.skip=true
```

**Comment by TylerJewell** [comment]
This is a fresh merge from this week.  @slemeur - we can do a quick fix for this please.

**Comment by ghost** [comment]
@TylerJewell https://raw.githubusercontent.com/eclipse/che/4.7.x/ide/che-core-ide-stacks/src/main/resources/images/type-bitnami.svg fixed in 4.7.x
